coral/src/app/pages/topics/index.tsx

```diff
@@ -1,12 +1,21 @@
-import BrowseTopics from "src/app/features/browse-topics/BrowseTopics";
 import { Flexbox, PageHeader } from "@aivenio/design-system";
+import add from "@aivenio/design-system/dist/src/icons/add";
 import AuthenticationRequiredBoundary from "src/app/components/AuthenticationRequiredBoundary";
+import BrowseTopics from "src/app/features/browse-topics/BrowseTopics";
 
 const Topics = () => {
   return (
     <AuthenticationRequiredBoundary>
       <Flexbox direction={"column"} rowGap={"l2"}>
-        <PageHeader title={"Browse all topics"} />
+        <PageHeader
+          title={"Browse all topics"}
+          primaryAction={{
+            text: "Request A New Topic",
+            // @TODO Replace by useNavigate once the request page is implemented in coral
+            onClick: () => (window.location.href = "/requestTopics"),
```

Comment:
Hm. This should be a link (looking like a button, for us right now ExternalButton woudl work) and not a button, then we wouldn't need to to this 🤔  Maybe that's something that should be raised with the Design System? 

Also I would like to throw in that we may want to consider some temporary tweaks for this button, because there are some easy to fix accessibility issues 🙈  But we can do that more globally (e.g. add a proper focus state etc) and pay attention that our own  theme solves the color issues (the contrast isn't enough)

Reply:
Ah yes, I simply followed the DS API for the `PageHeader` component! It should definitely be a link, or as you at least a link shaped to look like a button (`ExternalButton`).

I don't think there is a way top achieve this currently with the `PageHeader` component, so I will raise it with the DS team. I think the use-case of having an external link in that component is quite common and should be handled via an `external: boolean` option on `primaryAction` and `secondaryAction`. Like, the "Learn more" secondary action example in the docs seems like it would be an external link.

Reply:
button vs link - It's not a super big issue, as it's usable for all users, just semantics in that case. Updating that in the DS (if that's ok for DS) at some point, and all good :D (if react-router is fine with that, of course)

Color and focus issue is probably something more general for finetuning in the end!

Reply:
Buttons that should be links is a minor pet peeve of mine tbh haha! Buttons do things, links go places 😤 

But I agree that the color and focus states are bigger hindrance to usability.

Should it be raised as an issue in the DS repo, or a question in the channel, in your opinion?

Reply:
> Should it be raised as an issue in the DS repo, or a question in the channel, in your opinion?
The DS is aware of this, it's part of the discussion, so no need to drop it in. 

I would say we wait until all features are in for `/topics` and then have a discussions in general, as there are several places where we could improve functionality for users. 

There's [a ticket](https://github.com/aiven/klaw/issues/240) to collect these things (& to remind us about it!.

Reply:
I added the issues noted in this PR as a comment on the issue https://github.com/aiven/klaw/issues/240#issuecomment-1344161064